tests/wpt/metadata/encoding/textencoder-constructor-non-utf.html.ini

```diff
@@ -1,120 +1,3 @@
 [textencoder-constructor-non-utf.html]
```

Comment:
Please remove this file entirely, and squash the commits together. r=me with that done.